src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -1788,12 +1783,11 @@ impl<'a> QemuCmdLine<'a> {
 
 pub fn get_network_device(
     config: &HypervisorConfig,
-    dev_index: u64,
     host_dev_name: &str,
     guest_mac: Address,
 ) -> Result<(Netdev, DeviceVirtioNet)> {
     let mut netdev = Netdev::new(
-        &format!("network-{}", dev_index),
+        host_dev_name,
```

Comment:
I guess `host_dev_name` provides enough guarantees of uniqueness to be used here and it gives a hint for debugging, but I'm not big fan of silently confounding host device names and QEMU's netdev names. It might be confusing for new contributors. I'd personally keep a discriminator, e.g. something like :

```suggestion
        &format!("network-{}", host_dev_name),
```

Reply:
Fair enough, I'll make that change.